test/integration/catalog-management.v1.test.js

```diff
@@ -1,6 +1,5 @@
-/* eslint-disable no-console */
```

Comment:
This file is missing tests for deleteShareApprovalList() and getOfferingSourceArchive().